django/utils/timezone.py

```diff
@@ -295,13 +295,15 @@ def template_localtime(value, use_tz=None):
 
 # Utilities
 
-def localtime(value, timezone=None):
+def localtime(value=None, timezone=None):
     """
     Converts an aware datetime.datetime to local time.
 
     Local time is defined by the current time zone, unless another time zone
     is specified.
     """
+    if value is None:
+        value = _now_utc()
```

Comment:
Yes, this helper is no longer needed if we fallback from `USE_TZ` is `False`. I missed the test you mentioned, to realize `localtime()` was considered to work only when `USE_TZ` is `True`.